blockchain/README.md

```diff
@@ -0,0 +1,64 @@
+## Prerequisits (one time)
+
+# Requirements
+
+- Windows 10 with 64 bit 
+- Windows 11 with 64 bit operating system 
+
+# How to install on windows 
+
+- The first place to start is the official Docker website(https://www.docker.com/get-started/) from where  download Docker Desktop.
+- Run the downloaded installer 
+- Configuration
+    Mark both options true
+- Click Ok and wait a bit
+- Restart required
+- Docker automatically run Accept License Agreement
+- the Docker Desktop window will open. we have selected WSL 2 as our virtualization enginein configuration, we also need to install it. Don’t click Restart just yet!
```

Comment:
is WSL 2 needed? can they do it without this?

Reply:
in link which i mentioned as reference there it mentioned it's needed to install WSL 2